Components/Molecules/DashboardMetrics.js

```diff
@@ -23,12 +36,130 @@ const PercentageComplete = ({ goals }) => {
+  });
+  const timeStampsFiltered = filterTimeStampsForCadenceRelative({
+    timeStamps: timeStampsList,
+    time_interval,
+    step,
+  });
+  const count = timeStampsFiltered.length;
+  return count;
+};
+const PushesToday = ({ timeStamps }) => {
+  const count = PushesFromTimeRange({ timeStamps, time_interval: "day", step: 0 });
+  return <MetricNeomorph number={count} text={"Today"} />;
+};
+
+const PushesYesterday = ({ timeStamps }) => {
+  const count = PushesFromTimeRange({ timeStamps, time_interval: "day", step: 1 });
+  return <MetricNeomorph number={count} text={"Yest."} />;
+};
+
+const PushesBeforeYesterday = ({ timeStamps }) => {
+  const count = PushesFromTimeRange({ timeStamps, time_interval: "day", step: 2 });
+  return <MetricNeomorph number={count} text={"Prior Day"} />;
+};
+
+const PushesThisWeek = ({ timeStamps }) => {
+  const count = PushesFromTimeRange({ timeStamps, time_interval: "week", step: 0 });
+  return <MetricNeomorph number={count} text={"This Week"} />;
+};
+
+const PushesLastWeek = ({ timeStamps }) => {
+  const count = PushesFromTimeRange({ timeStamps, time_interval: "week", step: 1 });
+  return <MetricNeomorph number={count} text={"Last Week"} />;
+};
+
+const PushesLastWeekPrior = ({ timeStamps }) => {
+  const count = PushesFromTimeRange({ timeStamps, time_interval: "week", step: 2 });
+  return <MetricNeomorph number={count} text={"Prior Week"} />;
+};
+
+const PushesThisMonth = ({ timeStamps }) => {
+  const count = PushesFromTimeRange({ timeStamps, time_interval: "month", step: 0 });
+  return <MetricNeomorph number={count} text={"This Month"} />;
+};
+
+const PushesLastMonth = ({ timeStamps }) => {
+  const count = PushesFromTimeRange({ timeStamps, time_interval: "month", step: 1 });
+  return <MetricNeomorph number={count} text={"Last Month"} />;
+};
+
+const PushesLastMonthPrior = ({ timeStamps }) => {
+  const count = PushesFromTimeRange({ timeStamps, time_interval: "month", step: 2 });
+  return <MetricNeomorph number={count} text={"Prior Month"} />;
+};
+
 const DashboardMetrics = ({ goals, timeStamps }) => {
+  const [items, updateItems] = useState([]);
+  useEffect(() => {
+    updateItems([
+      <DueGoals goals={goals} />,
+      <CompleteGoals goals={goals} />,
+      <PercentageComplete goals={goals} />,
+      <TotalPushes timeStamps={timeStamps} />,
+      <LongestStreak goals={goals} />,
+      <OldestTask goals={goals} />,
+      <PushesToday timeStamps={timeStamps} />,
+      <PushesYesterday timeStamps={timeStamps} />,
+      <PushesBeforeYesterday timeStamps={timeStamps} />,
+      <PushesThisWeek timeStamps={timeStamps} />,
+      <PushesLastWeek timeStamps={timeStamps} />,
+      <PushesLastWeekPrior timeStamps={timeStamps} />,
+      <PushesThisMonth timeStamps={timeStamps} />,
+      <PushesLastMonth timeStamps={timeStamps} />,
+      <PushesLastMonthPrior timeStamps={timeStamps} />,
+
+      // <PercentageComplete goals={goals} />,
+      // <AverageAge goals={goals} />,
```

Comment:
delete if not planning to use later. Also, seeing many un-used imports and variables here and there

Reply:
true - ill clean up